packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -341,10 +353,10 @@ export class HttpPlugin extends BasePlugin {
    * @param options The arguments to the original function.
    */
   private getMakeRequestTraceFunction(
-      // tslint:disable-next-line:no-any
       request: httpModule.ClientRequest, options: httpModule.RequestOptions,
       plugin: HttpPlugin): Func<httpModule.ClientRequest> {
     return (span: Span): httpModule.ClientRequest => {
+      const startTime = Date.now();
```

Comment:
Could this be a high-accuracy timestamp? I believe some Node versions support `performance.now()`, which would be an easy swap in, or else you could use `process.hrtime` and do some conversion / use a utility.

Reply:
For stats recoding, we want to record ```ms``` level granularity. Looks like Date.now() gives us that. WDYT?

Reply:
Yes, that's fine. Didn't realize the stats were only at `ms` granularity.